x/bank/client/testutil/grpc.go

```diff
@@ -1,286 +0,0 @@
-package testutil
-
-import (
-	"fmt"
-
-	"cosmossdk.io/math"
-	"github.com/gogo/protobuf/proto"
-
-	"github.com/cosmos/cosmos-sdk/testutil"
-	"github.com/cosmos/cosmos-sdk/testutil/rest"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	grpctypes "github.com/cosmos/cosmos-sdk/types/grpc"
-	"github.com/cosmos/cosmos-sdk/types/query"
-	"github.com/cosmos/cosmos-sdk/x/bank/types"
-)
-
-func (s *IntegrationTestSuite) TestTotalSupplyGRPCHandler() {
```

Comment:
Should we not keep these in `tests/e2e/bank` prior to refactoring, then? It's being used in `tests/e2e/bank/cli_test.go`

Reply:
No.

Reply:
That's what I mean: to move temporarily these deleted tests files in /tests/e2e/bank until we refactor them properly :)